### ISSUE TITLE
Wish to publish an EXE file

### ISSUE THREAD

**Issue by maomingyang1314** [madMAx43v3r/chia-plotter#30]
Dear developer, I hope you can compile the exe file that can run on windows

**Comment by msn1983aa** [comment]
yes,we need an exe file

**Comment by 122474363** [comment]
yes,we need an exe file

**Comment by sigi77777** [comment]
please do not post any executable (especially for windows), in this way we can prevent the new plotter from spreading accross non technical people.

**Comment by maomingyang1314** [comment]
Indeed, I am also aware of this problem, let's just a few of us party
